pkg/config/validate.go

```diff
@@ -175,6 +175,9 @@ func validateModule(p modulePath, m Module, bp Blueprint) error {
 	if m.ID == "" {
 		errs.At(p.ID, fmt.Errorf(errorMessages["emptyID"]))
 	}
+	if m.ID == "vars" { // invalid module ID
+		errs.At(p.ID, errors.New("can not use vars as module id"))
```

Comment:
perhaps phrase this error a little better?  For instance, what about:
"module id cannot be 'vars'".

Reply:
Done.